public/app/plugins/datasource/loki/components/monaco-query-field/monaco-completion-provider/completions.test.ts

```diff
@@ -814,6 +724,21 @@ describe('IN_LOGFMT completions', () => {
     `);
   });
 
+  it('autocompleting logfmt labels should correctly add trailing commas', async () => {
```

Comment:
```
it('autocompleting logfmt with flags should only offer labels when the user has a trailing space', async () => {
    const situation: Situation = {
      type: 'IN_LOGFMT',
      logQuery: `{job="grafana"} | logfmt label3 `,
      flags: true,
      otherLabels: ['label1'],
    };

    expect(await getCompletions(situation, completionProvider)).toMatchInlineSnapshot(`
      [
        {
          "insertText": "label2",
          "label": "label2",
          "triggerOnInsert": false,
          "type": "LABEL_NAME",
        },
      ]
    `);
  });
```
Fails on this branch if you just delete the comma at the end. Can we add unit tests for all of the test cases we've discovered so far?

Reply:
I forgot to update the test titles, so just synced them and added a few more. If there's anything else you think we have to test just let me know.